src/inscription.rs

```diff
@@ -231,7 +260,19 @@ impl<'a> InscriptionParser<'a> {
         }
       }
 
-      return Ok(Some(Inscription { body, content_type }));
+      return Ok(Some(Inscription {
+        body,
+        content_type,
+        parent: match parent {
+          None => None,
+          Some(bytes) => {
+            if bytes.len() != 36 {
+              return Err(InscriptionError::InvalidInscription)
```

Comment:
Might just ignore and return `None` here instead of throwing Error.

Reply:
Yep, this part unfinished, but you're right, we should gracefully ignore malformed inscriptions.